lsp-mode.el

```diff
@@ -4245,6 +4307,26 @@ it has to calculate identation based on SRC block position."
             (with-no-warnings (undo-amalgamate-change-group change-group)))
           (progress-reporter-done reporter))))))
 
+(defun lsp--create-apply-text-edits-handlers ()
+  "Create (handler cancel-handler) for applying text edits in async request.
+Only works when mode is 'tick or 'alive."
+  (let* (first-edited
+         (func (lambda (start &rest _)
+                 (setq first-edited (if first-edited
+                                        (min start first-edited)
+                                      start)))))
+    (add-hook 'before-change-functions func nil t)
+    (list
+     (lambda (edits)
+       (unless (and first-edited
```

Comment:
can you add some kind of message that the additional edits will be skipped? Not 100% sure if it should be info/warn or lsp-log

Reply:
added as warn